microservice/settings/src/main/java/com/cumulocity/microservice/settings/service/impl/MicroserviceSettingsServiceImpl.java

```diff
@@ -83,6 +83,12 @@ public String get(@NonNull String key) {
         return getAll().get(key);
     }
 
+    @Override
+    public void evict(){
+        final MicroserviceCredentials credentials = getCurrentCredentials();
```

Comment:
unsure it's in context, throw exception otherwise. Let's avoid accidental evict for management.